Examples/Tests/PythonWrappers/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,291 @@
+        ax.annotate('PML', xy = (nx//2,nz+nzpml//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (-nxpml-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx+nxpml+nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,-nzpml-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('PML ghost', xy = (nx//2,nz+nzpml+nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        # Set extent and sliced data
+        extent = np.array([-nxg-nxpml, nx+nxpml+nxg, -nzg-nzpml, nz+nzpml+nzg])
+    else:
+        # Draw ghost regions
+        ax.axvline(x = 0 , linewidth = lw, linestyle = ls)
+        ax.axvline(x = nx, linewidth = lw, linestyle = ls)
+        ax.axhline(y = 0 , linewidth = lw, linestyle = ls)
+        ax.axhline(y = nz, linewidth = lw, linestyle = ls)
+        # Annotations
+        ax.annotate('ghost', xy = (-nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx+nxg//2,nz//2), rotation = 'vertical', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx//2,-nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        ax.annotate('ghost', xy = (nx//2,nz+nzg//2), rotation = 'horizontal', ha = 'center', va = 'center')
+        # Set extent and sliced data
+        extent = np.array([-nxg, nx+nxg, -nzg, nz+nzg])
+    X = data[:,:].transpose()
+    # Min and max for colorbar
+    vmin, vmax = compute_minmax(X)
+    # Display data as image
+    im = ax.imshow(X = X, origin = 'lower', extent = extent, vmin = vmin, vmax = vmax, cmap = 'seismic')
+    # Add colorbar to plot
+    fig.colorbar(im, cax = cax)
+    # Set label for x- and y-axis, set title
+    ax.set_xlabel('x')
+    ax.set_ylabel('z')
+    ax.set_title(title)
+    # Set plot title
+    suptitle = 'PML in (x,z), 4 grids 64 x 64'
+    plt.suptitle(suptitle)
+    # Save figure
+    figname = 'figure_' + name + '.png'
+    fig.savefig(figname, dpi = 100)
+
+# Initialize fields data (unit pulse) and apply smoothing
+def init_data(data):
+    impulse_1d = np.array([1./4., 1./2., 1./4.])
+    impulse = np.outer(impulse_1d, impulse_1d)
+    data[nx//2-1:nx//2+2,nz//2-1:nz//2+2] = impulse
+
+# Initialize inputs and WarpX instance
+sim.initialize_inputs()
+sim.initialize_warpx()
+
+# Get fields data using Python wrappers
+import pywarpx.fields as pwxf
```

Comment:
Let's move this import to the top

Reply:
If I try to move this `import` at the beginning of the file, I actually get a crash (discussed on Slack).

Reply:
Jeeze, for the record it's:
```
Invalid MIT-MAGIC-COOKIE-1 keyTraceback (most recent call last):
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/_libwarpx.py", line 81, in <module>
    libwarpx = ctypes.CDLL(os.path.join(_get_package_root(), libname))
  File "/usr/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/edoardo/.local/lib/python3.8/site-packages/pywarpx/libwarpx.3d.so: cannot open shared object file: No such file or directory
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "PICMI_inputs_2d.py", line 5, in <module>
    import pywarpx.fields as pwxf
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/fields.py", line 24, in <module>
    from . import _libwarpx
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/_libwarpx.py", line 85, in <module>
    raise Exception(f'"{libname}" was not installed. Installation instructions can be found here https://warpx.readthedocs.io/en/latest/install/users.html') from e
Exception: "libwarpx.3d.so" was not installed. Installation instructions can be found here https://warpx.readthedocs.io/en/latest/install/users.html
Traceback (most recent call last):
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/_libwarpx.py", line 81, in <module>
    libwarpx = ctypes.CDLL(os.path.join(_get_package_root(), libname))
  File "/usr/lib/python3.8/ctypes/__init__.py", line 373, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /home/edoardo/.local/lib/python3.8/site-packages/pywarpx/libwarpx.3d.so: cannot open shared object file: No such file or directory
The above exception was the direct cause of the following exception:
Traceback (most recent call last):
  File "PICMI_inputs_2d.py", line 5, in <module>
    import pywarpx.fields as pwxf
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/fields.py", line 24, in <module>
    from . import _libwarpx
  File "/home/edoardo/.local/lib/python3.8/site-packages/pywarpx/_libwarpx.py", line 85, in <module>
    raise Exception(f'"{libname}" was not installed. Installation instructions can be found here https://warpx.readthedocs.io/en/latest/install/users.html') from e
Exception: "libwarpx.3d.so" was not installed. Installation instructions can be found here https://warpx.readthedocs.io/en/latest/install/users.html
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpiexec detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:
  Process name: [[31374,1],1]
  Exit code:    1
--------------------------------------------------------------------------
```

Haven't seen that, but it's not good.

Reply:
Needs an issue :)

Reply:
Here it is: #2633.